types/big.go

```diff
@@ -23,11 +23,11 @@ func (i *BigInt) UnmarshalText(data []byte) error {
 }
 
 func (i *BigInt) GobEncode() ([]byte, error) {
-	return i.ToInt().GobEncode()
+	return i.ToStdBigInt().GobEncode()
```

Comment:
This naming change is out of the scope for this PR :(

Reply:
Well, the scope of this PR was to quickly fix the API retuning empty results, and we've spent 1 week iterating over some things that are not in the scope of this PR. 
I agree that this naming change could be included in another PR, but given how this PR evolved, the change does not hurt anyone and IMHO can be included here.
At this point, I prefer not to roll back the changes.

Reply:
Ok. But a separated commit with this change would have been appreciated.

Reply:
Gotcha, I will do it better next time.